### PR TITLE
RAXAJSS-109 liquibase.xml edited

### PR DESCRIPTION
**Pull request by TarangMahajan**

### CODE REVIEW

**Comment by nathanleiby** [comment]
Looks good - seems to meet requirements specified in the ticket

File: api/src/main/resources/liquibase.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+		</preConditions>
+		<comment>
+			Schema for raxacore_patient_list table
+		</comment>
+		<createTable tableName="raxacore_patient_list">
+			<column name="patient_list_id" type="char(38)"/>
+			<column name="name" type="varchar(255)"/>
+			<column name="description" type="varchar(2000)">
+			<column name="search_query" type="varchar(1000)">
+			<column name="uuid" type="int">
```
Comment (sunbiz) [inline api/src/main/resources/liquibase.xml]: Like I mentioned in the review on ur branch, uuid should be char(38)
Rest all looks fine. Can you make that change and send a pull request

**Comment by TarangMahajan** [comment]
is it fine now sir ?

**Comment by nathanleiby** [comment]
Looks good

File: api/src/main/resources/liquibase.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  	
+	<changeSet id="raxacore_patient_list" author="tarang">
+		<preConditions onFail="MARK_RAN">
+			<not><tableExists tableName="raxacore_patient_list"/></not>
+		</preConditions>
+		<comment>
+			Schema for raxacore_patient_list table
+		</comment>
+		<createTable tableName="raxacore_patient_list">
+			<column name="patient_list_id" type="char(38)"/>
```
Comment (johnstoecker) [inline api/src/main/resources/liquibase.xml]: The type here should be int

**Comment by TarangMahajan** [comment]
done.

**Pull request merged**